docs/IslandPruning.md

```diff
@@ -0,0 +1,89 @@
+# Street graph pruning
+
+Street graph created from OpenStreetMap data has sections which cannot be reached with some or all traverse modes. For example,
+true geographic islands are usually disconnected from the main graph. Street network of such islands should be kept so that routing
```

Comment:
```suggestion
true geographic islands are usually disconnected from the main graph. The street network of such islands should be kept so that routing
```